Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinationCertificate.kt

```diff
@@ -25,6 +25,7 @@ interface VaccinationCertificate : CwaCovidCertificate {
     override val rawCertificate: VaccinationDccV1
 
     val isSeriesCompletingShot get() = doseNumber >= totalSeriesOfDoses
+    val isBooster get() = doseNumber > totalSeriesOfDoses
```

Comment:
This could be misleading when we apply it to all VC. According to the EU spec
>Boosters will be recorded as follows:
>- 3/3 for a booster dose following a primary 2-dose vaccination series.
>- 2/1 for a booster dose following a single-dose vaccination or a one dose of a 2-dose vaccine administered to a recovered person.

or did something change?

Reply:
Alright, I will remove it from here and just add a comparison in `VaccinatedPerson`